### PR TITLE
OPENINSAI: SWI-4551 - Prepare saivlan.py file for upstreaming

### PR DESCRIPTION
**Pull request by pkochmanska**

### CODE REVIEW

**Review by ravi861 — APPROVED** [review]
Good to go

File: ptf/saivlan.py [hunk 1]
```diff
@@ -0,0 +1,2553 @@
...
+        ])
+        test.fail(
+            "Did not receive expected packets on any of {} for device {}. \n{}"
+            .format(ports, device_number, failure_report))
+
+    test.assertTrue(
+        pkt_cnt >= len(ports),
+        "Did not receive pkt on one of ports %r for device %d" %
+        (ports, device_number))
+    return rcv_idx
```
Comment (aljer) [inline ptf/saivlan.py]: This helper part should be removed. You should be able to replace verify_any_packet_on_ports_list() function with verify_packets_on_multiple_port_lists() after refactoring messy code between 1430-1517

File: ptf/saivlan.py [hunk 2]
```diff
@@ -0,0 +1,2553 @@
+#  Copyright 2020-present Intel Corporation.
```
Comment (aljer) [inline ptf/saivlan.py]: 2021

**Review by aljer — APPROVED** [review]
Good job! lgtm

**Pull request closed without merging**